### DISCUSSION TITLE
Circular instructions when configuring custom wildcard subdomain

### DISCUSSION THREAD

**Discussion by tim-codes** [vercel/community#2625] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/domains/

### Steps to Reproduce

1. Configure wildcard subdomain using nameservers method
-- added `*.preview.mydomain.io` in Vercel UI
-- configured nameservers in my DNS provider -> NS record `preview.mydomain.io ns1.vercel-dns.com. ns1.vercel-dns.com.`
![CleanShot 2023-05-27 at 01 52 37@2x](https://github.com/vercel/community/assets/47059022/8403d397-b258-4da5-a381-4a358634ef34)
-- so far so good 👍🏻 

2. Follow instructions in the Vercel UI to add CNAME record
![CleanShot 2023-05-27 at 01 48 30@2x](https://github.com/vercel/community/assets/47059022/ce9d5107-7616-4c78-bfcc-9fbb60bd7ea1)
-- click the "add" link which takes me to Vercel DNS records
-- add the CNAME record as instructed
![CleanShot 2023-05-27 at 01 51 06@2x](https://github.com/vercel/community/assets/47059022/3d8df394-ff06-4f90-bda0-2c4fbeb0b474)

3. returning to the previous screen, now the UI instructs that the new record is a conflict and should change it for an A record
![CleanShot 2023-05-27 at 01 53 28@2x](https://github.com/vercel/community/assets/47059022/157bbf83-501b-4276-82f4-823734ec188e)

4. swapping to A record as instructed returns back to (2) state, asking to swap A record for CNAME record
🤯

I can't find any reference in the documentation that would clear up the correct configuration here, but the circular instructions seem to indicate a bug.

Any help appreciated! 🙏

**Comment by amyegan** [comment]
Sorry you ran into this problem, @tim-codes. If you share the domain, we can look into the logs for more info on what is going wrong.

**Reply by tim-codes** [reply]
Thanks @amyegan, the domain is \

**Reply by tim-codes** [reply]
Hey @amyegan do you have a rough idea of when this might get looked at? If it will be a while, I might look at another approach that doesn't use Vercel DNS.

**Reply by amyegan** [reply]
I tried a couple of different ways and have not been able to replicate the issue you're seeing. I reached out to our network team and am waiting to get more info as they investigate this bug.

**Reply by tim-codes** [reply]
Thanks @amyegan, I've tried with a second domain *.preview.contentmesh.io, and I'm getting the same behaviour

**Reply by tim-codes** [reply]
@amyegan are you able to clarify what should be the correct configuration? Eg should it be the A record or the CNAME record?

**Reply by tim-codes** [reply]
Hey @amyegan it's been a couple of weeks now, I'm wondering if the team managed to take a look at this issue yet. I'm keen to get this DNS set up on this project 🙏

**Reply by amyegan** [reply]
Sorry for the delay, @tim-codes. The engineers are looking at it, but it's a tricky problem to solve since we have not been able to reproduce the issue on other projects. 

I appreciate your patience as we work out what is going wrong for your configuration. If there is any other configuration info you're able share, I am happy to pass it along to our network team 🙇

**Reply by tim-codes** [reply]
@amyegan I'm happy to try a few more things my side but there isn't any reference configuration in the documentation. Maybe your team could provide an example of a correct configuration using subdomain wildcard delegation and I can try to troubleshoot against that? Then I can rule out user error.

**Reply by smaeda-ks** [reply]
@tim-codes We're fixing this. Will update you soon

**Reply by tim-codes** [reply]
@smaeda-ks awesome news :) Thanks so much

**Reply by amyegan** [reply]
The PR was rolled out so you will see the correct error message if there are any problems with DNS records. Please let us know if you continue to have problems with circular instructions 🙏
